RxMarbles/HelpViewController.swift

```diff
@@ -703,122 +671,140 @@ class HelpViewController: AnimatedPagingScrollViewController, UITextViewDelegate
         let containerWidth = container.widthAnchor.constraint(equalToConstant: 300)
         contentView.addConstraints([containerTop, containerBottom, containerWidth])
         
-        let manyLikeLabel = UILabel()
-        
-        let erikMeijerTwitter = RxMarbles.Image.twitter.imageView()
-        let erikMeijerTextView = UITextView()
-        
-        let krunoslavZaherTwitter = RxMarbles.Image.twitter.imageView()
-        let krunoslavZaherTextView = UITextView()
-        
-        let rxSwiftLabel = UILabel()
-        let githubButton = UIButton(type: .custom)
-        let andTwitterLabel = UILabel()
-        let rxSwiftTwitterButton = UIButton()
-        let alasLabel = UILabel()
+		let manyLikeLabel: UILabel = {
+			let label = UILabel()
+			label.text = "Many ❤️👍🍻👋 to"
+			label.font = Font.code(.malayalamSangamMN, size: 18)
+			label.translatesAutoresizingMaskIntoConstraints = false
+			return label
+		}()
+		let erikMeijerTwitter: UIImageView = {
```

Comment:
empty lines between the lets - since they're functions at this point, makes it a tad hard to read